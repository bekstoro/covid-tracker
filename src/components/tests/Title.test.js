import React from 'react';

import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import Title from '../Title'

let component;

describe('Title component', () => {
  beforeEach(() => {
    component = render(<Title>Test</Title>);
  });
  
  it('should render correctly', () => {
    expect(component.asFragment()).toMatchSnapshot();
  });
  
  it('should have title text', () => {
    expect(component.getByText(/Test/i)).toBeInTheDocument();
  });
});
