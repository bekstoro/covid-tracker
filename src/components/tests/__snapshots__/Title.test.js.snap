// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render correctly 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
  >
    Test
  </h2>
</DocumentFragment>
`;
